partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -52,8 +52,37 @@ statement::[
   query::{
     root: rex,
   },
+  d_d_l::{
+    op: ddl_op,
+  },
+]
+
+// DDL is an experimental feature.
+// We are actively working on finializing the semantics of DDL operation.
+// As of today,
+// the public plan aims to expose as minium implementation details as possible to avoid breaking changes in the feature.
+ddl_op::[
+  create_table::{
+    // Table Name, unresolved
+    name: identifier,
```

Comment:
I would think this would be fully-resolved for the public plan.

Reply:
I want to delegate the validation logic to connector at the moment, until we fully flush out the unix style path resolution for create command. 

At the moment, the connector shall receive a identifier, and it needs to implement its own logic as of where to persist the table. 

This is, the planner at this stage will not complain about problems like `Table already exists`, `schema not exist`, etc.

The comment is confusing, I will change it.

Reply:
I'm not entirely convinced by this since the connector is distinct from our public plan representation. In the connector (metadata), it doesn't necessarily need to check for whether the table already exists, but, the plan should be unambiguous. The executor can complain about the table already existing at the runtime, but still, it needs to know where to place the table (in an abstract sense) and in my opinion shouldn't leave it up to the evaluator's discretion.

If there is a concern regarding future compatibility, could we only allow creation of tables in the current schema? AKA, non-qualified table names. This would comply with our intention of the "relative path" as well as SQL:1999's semantics. Also, all of the tests are written this way currently. And, this would result in a disambiguated plan.

Reply:
And, on top of this, we gain control over our language. If we propagate case insensitivity and non-qualified identifiers in the plan (say table `tbl`), one evaluator might create a table named `"TBL"` in the current schema, and another evaluator could create the table named `"tbl"` in some other schema. In other words, by propagating the ambiguity, we rely on our consumers to know the internal, subtle details of the PartiQL language, and our semantics are not enforced.